layouts/default.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <v-app id="app">
-    <SbcHeader class="sbc-header" />
+    <SbcHeader class="sbc-header" :in-auth="false" />
+
+    <!-- Alert banner -->
+    <v-alert
+      v-if="bannerText"
+      tile dense
+      icon=" "
```

Comment:
Did you mean to specify an icon here? Can you just leave this property out?

Reply:
If don't use v-html then have to put the icon value like that. I tried couple way to remove v-html but not success or too complicated. I gave it up. 

I will remove the icon property.